modules/network-observability-configuring-options-overview.adoc

```diff
@@ -7,10 +7,21 @@
 = Configuring advanced options for the Overview view
 You can customize the graphical view by using advanced options. To access the advanced options, click *Show advanced options*.You can configure the details in the graph by using the *Display options* drop-down menu. The options available are:
 
-* *Metric type*: The metrics to be shown in *Bytes* or *Packets*. The default value is *Bytes*.
 * *Scope*: To select the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, service, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
```

Comment:
Sorry, again I noticed a slight issue in the preview:

```suggestion
* *Scope*: Select the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, service, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
```